pkg/rules/register.go

```diff
@@ -0,0 +1,24 @@
+package rules
+
+import (
+	"github.com/aquasecurity/defsec/pkg/framework"
+	"github.com/aquasecurity/defsec/pkg/scan"
+	"github.com/aquasecurity/trivy-iac/internal/rules"
+	"github.com/aquasecurity/trivy-policies/pkg/types"
+)
+
+func Register(rule scan.Rule, f scan.CheckFunc) types.RegisteredRule {
```

Comment:
This function is used to register Go rules. For example, https://github.com/aquasecurity/trivy-policies/blob/main/rules/cloud/policies/aws/s3/block_public_acls.go#L11.

Reply:
> Maybe we should make Go rules in trivy-policies public and register them with trivy-iac?

Yes that's the idea. I was planning for that happen via https://github.com/aquasecurity/trivy-iac/blob/335c78327d3e3062b86e488b191adf13ded61c21/pkg/rego/embed.go#L9 where the rules are imported.

But it seems a couple of tests are failing at the moment. Upon investigating them it seems they don't seem to be registered Go rules, so an import somewhere is missing (which would trigger the registration of Go rules).

Reply:
@simar7  When importing rules, they are saved in the rules registry, which is located in trivy-policies. Therefore, importing rules is not enough. https://github.com/aquasecurity/trivy-policies/blob/main/internal/rules/register.go#L22